mmcv/utils/config.py

```diff
@@ -555,20 +555,27 @@ def __setstate__(self, state):
         super(Config, self).__setattr__('_text', _text)
 
     def dump(self, file=None):
+        """
+            If a file argument is given, saves the config to that file using the format defined by the file argument extension.
+            Otherwise, returns a string representing the config.
+            The formatting of this returned string is defined by the extension of self.filename.
+            If self.filename is not defined, returns a string representation of a dict (lowercased and using ' for strings).
+        """
+        import mmcv
```

Comment:
Thanks for your docstring! It could be better to add an overall description and remove some indents. Besides, The `If a file argument is given.....` is too long, maybe we could start a new line~
```suggestion
        """Dump config to file or return config text.
        
        If a file argument is given, saves the config to that file using the format defined by the file argument extension.
        Otherwise, returns a string representing the config.
        The formatting of this returned string is defined by the extension of self.filename.
        If self.filename is not defined, returns a string representation of a dict (lowercased and using ' for strings).
        """
        import mmcv
```